spec/features/bookings_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe 'bookings data' do
+  before(:each) do
+    login_as FactoryGirl.create(:user)
+  end
+
+  it 'shows count of current and inactive bookings from past week', js: true do
```

Comment:
Don't think this spec requires `js: true` - can we remove that to speed up our test suite?